x/staging/subspaces/keeper/keeper.go

```diff
@@ -197,3 +197,22 @@ func (k Keeper) UnbanUserInSubspace(ctx sdk.Context, subspaceID, user, admin str
 	store.Delete(types.SubspaceBannedUserKey(subspaceID, user))
 	return nil
 }
+
+// CheckSubspaceUserPermission checks the permission of the given user inside the subspace with the
+// given id to make sure they are able to perform operations inside it
+func (k Keeper) CheckSubspaceUserPermission(ctx sdk.Context, subspaceID string, user string) error {
+	subspace, found := k.GetSubspace(ctx, subspaceID)
+	if !found {
+		return sdkerrors.Wrapf(types.ErrInvalidSubspaceID, "the subspace %s doesn't exist", subspaceID)
```

Comment:
I think we can avoid being more specific. The error should already tell the user what's wrong so we can remove the custom string and just reference the subspace id instead

```suggestion
		return sdkerrors.Wrap(types.ErrInvalidSubspaceID, subspaceID)
```